source/mainnet/net/desktop-wallet/update-application.rst

```diff
@@ -0,0 +1,30 @@
+.. _update-application:
+
+===========================
+Updating the desktop wallet
```

Comment:
```suggestion
Update the desktop wallet
```

Reply:
I don't think I get this. In my opinion "Update ..." sounds like a request to the reader, where as "Updating ..." sounds more like an introduction to a topic?

Reply:
It's not standard in documentation anymore to start with an -ing verb or a gerund (it used to be a couple of years ago). It's more common to use the infinitive. Alternatively, you could write Automatic updates of the Desktop Wallet or How to update the Desktop Wallet.

Reply:
I'll leave it for you to decide, so I'll apply the suggestion 🙂 thanks for clarifying.